mmv1/api/type.rb

```diff
@@ -664,6 +783,20 @@ def nested_properties
       end
     end
 
+    # Support for schema ValidateFunc functionality.
+    class Validation < Object
```

Comment:
non-blocking note: There don't appear to be any users of this yet, although excluding the configuration and doing it piecemeal would be undesirable. This corresponds to https://github.com/GoogleCloudPlatform/magic-modules/blob/8a5b9c547fb4febbcb43ad2d615efb8c1df3e0c0/mmv1/overrides/terraform/property_override.rb#L19-L31